tests/tests_twobody/test_propagation.py

```diff
@@ -39,7 +39,7 @@
 from poliastro.util import norm
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
Excellent!

Reply:
Thank you very much :)

Reply:
Just a question... Should I squash the commits or is it alright? :thinking:

Reply:
This is perfect! Each commit for a different thing